tardis/model/base.py

```diff
@@ -581,12 +582,18 @@ def from_csvy(cls, config):
             validate_dict(csvy_model_config, schemapath=csvy_schema_file)
         )
 
+        nuclides_hyphenless = np.char.replace(DEFAULTDATA.nuclides, "-", "")
```

Comment:
You created this variable in another place too. It would be better defined in a utility location for access in multiple locations.